localization/english/wc_traits_l_english.yml

```diff
@@ -498,4 +498,8 @@
 
  trait_creature_nightborne:0 "Nightborne"
  trait_creature_nightborne_desc:0 "The Nightborne are a mystic race of elves originating from Suramar. Since the destruction of the Well of Eternity, the isolated Nightborne has evolved to become a new race thanks to the power of the Nightwell. Like other elves they require sustenance in the form of magic. Because of this the Nightborne employ the power of the Nightwell in the form of Arcwine to sustain themselves. The Nightborne remain the only elven race to maintain the legacy of the Kaldorei empire, sustaining the advancement of magic long after Azshara fell."
- trait_creature_nightborne_character_desc:0 "This character is a Nighborne elf. $trait_creature_nightborne_desc$"
\ No newline at end of file
+ trait_creature_nightborne_character_desc:0 "This character is a Nighborne elf. $trait_creature_nightborne_desc$"
+
+ trait_creature_highmountain_tauren:0 "Highmountain Tauren"
+ trait_creature_highmountain_tauren_desc:0 "The Highmountain Tauren are an offshoot of Tauren descended from Huln Highmountain, who was an ally of the Kaldorei Resistance during the War of the Ancients. During the war, Huln saved Eche'ro, a moose favored by Malorne and was in turn granted a blessing from Cenarius. The blessing granted them the Horns of Eche'ro: moose-like antlers, as well as a minor boost to their lifespan."
```

Comment:
I wouldn't mention lifespan because... I feel like the desc is narrative, so we shouldn't use game terms there.